### DISCUSSION TITLE
Is it possible to show 'Empty' filter box in the filters web part?

### DISCUSSION THREAD

**Discussion by JoeAyre** [microsoft-search/pnp-modern-search#3886] [Ideas]
Hi all,

Just wondering if anyone knows whether it is possible to configure the filters web part to include a checkbox for items that **not** tagged with the corresponding filter field.
For example, consider this scenario:

- I have a document library containing 100 documents.
- The library has two columns: 'Colour' (choice column) and Region (choice column).
- The search results web part is configured to bring back all 100 documents.
- I have a filters web part that has two filter fields: 'Colour' and 'Region' using the 'Checkbox' template, with the operator between values set to 'OR' for both filter fields.
- The filters web part 'Operator to use between filters' option is set to AND
- I want to filter the results to show all documents that have been tagged with Colour = Red, but have **not** been tagged with 'Region'.

In native SharePoint lirbary filtering, the option to select 'Empty' is available (see image below), but I can't see this option in the filters web part. Does anyone have any advice on how this could be achieved?

![image](https://github.com/user-attachments/assets/2a554807-26b6-48ee-bc4f-a7c477e02f37)

Thanks,
Joe

**Comment by kasperbolarsen** [comment]
As a rule of thumb Search can't show "is null" values. In this case I would recommend that you change the two fields to have a "not decided yet" default value

**Reply by wobba** [reply]
For text values we can make this happen, ref https://www.techmikael.com/2023/04/filter-on-managed-properties-in-search.html. The filter FQL would then be something like `region:not("*")`.
